drivers/performance_measurement/spec/models/performance_measurement/pm_spm_alignment_spec.rb

```diff
@@ -644,4 +644,267 @@ def setup_reports(projects:)
       expect(pm_count).to eq(2) # 2 exits (client 2 from ES and client 4 from RRH)
     end
   end
+
+  describe 'result_calculation#client_sum method' do
+    include_context 'SPM test setup'
+
+    before do
+      # Create projects
+      @es_project = create_project(project_type: 0) # ES-EE
+      @th_project = create_project(project_type: 2) # TH
+
+      # Create two clients
+      @client1 = create_client_with_warehouse_link
+      @client2 = create_client_with_warehouse_link
+
+      # Create enrollments for the clients
+      create_enrollment(
+        client: @client1,
+        project: @es_project,
+        entry_date: test_start_date + 10.days,
+        exit_date: test_start_date + 40.days,
+      )
+
+      create_enrollment(
+        client: @client2,
+        project: @th_project,
+        entry_date: test_start_date + 50.days,
+        exit_date: test_start_date + 80.days,
+      )
+
+      # Setup report with these projects
+      @spm_report, @pm_report = setup_reports(projects: [@es_project, @th_project])
+
+      # Add different values for days homeless to demonstrate the issue
+      @client_record1 = @pm_report.clients.find_by(client_id: @client1.destination_client.id)
+      @client_record2 = @pm_report.clients.find_by(client_id: @client2.destination_client.id)
+
+      # Use the same value for both clients to demonstrate the issue with DISTINCT
+      @days_homeless_value = 30
+      @client_record1.update(reporting_days_homeless_es_sh_th: @days_homeless_value)
+      @client_record2.update(reporting_days_homeless_es_sh_th: @days_homeless_value)
+
+      # Create client_projects records
+      PerformanceMeasurement::ClientProject.create!(
+        report_id: @pm_report.id,
+        client_id: @client1.id,
+        project_id: @es_project.id,
+        for_question: :days_homeless_es_sh_th,
+        period: :reporting,
+      )
+
+      PerformanceMeasurement::ClientProject.create!(
+        report_id: @pm_report.id,
+        client_id: @client2.id,
+        project_id: @th_project.id,
+        for_question: :days_homeless_es_sh_th,
+        period: :reporting,
+      )
+    end
+
+    it 'should correctly sum values from different clients with the same value' do
+      # Get the result from client_sum for the days_homeless_es_sh_th field
+      sum_result = @pm_report.client_sum(:days_homeless_es_sh_th, :reporting)
+
+      # What we should get is the sum of all client values, even if they're the same
+      expected_result = @days_homeless_value * 2 # 60
+
+      # Check that we're getting the correct sum (both clients' values)
+      expect(sum_result).to eq(expected_result),
```

Comment:
Is it worth adding a test that worked the old way to confirm it returns a different result (essentially to confirm we didn't just accidentally get the right result?)